src/libraries/System.Drawing.Common/src/System.Drawing.Common.csproj

```diff
@@ -1,11 +1,10 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <DefineConstants>$(DefineConstants);DRAWING_NAMESPACE</DefineConstants>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <WarningsNotAsErrors>CS0618</WarningsNotAsErrors>
-    <DefineConstants Condition="'$(TargetsWindows)' == 'true'">$(DefineConstants);FEATURE_WINDOWS_SYSTEM_COLORS;FEATURE_SYSTEM_EVENTS</DefineConstants>
     <IsPartialFacadeAssembly>true</IsPartialFacadeAssembly>
-    <TargetFrameworks>$(NetCoreAppCurrent)-windows;$(NetCoreAppCurrent)-Unix;$(NetCoreAppCurrent);$(NetCoreAppMinimum)-windows;$(NetCoreAppMinimum)-Unix;$(NetCoreAppMinimum);netstandard2.0;$(NetFrameworkMinimum)</TargetFrameworks>
+    <TargetFrameworks>$(NetCoreAppCurrent)-windows;$(NetCoreAppCurrent);$(NetCoreAppMinimum)-windows;$(NetCoreAppMinimum);netstandard2.0;$(NetFrameworkMinimum)</TargetFrameworks>
```

Comment:
This will cause System.Drawing.Common to throw PlatformNotSupportedException on non Windows on any code path. Whereas now, it is throwing only if libgdiplus is tried to load. @ericstj and I chatted about this and thought this might break a very edge scenario where something can be used without loading libgdiplus at all and some library using that scenario will now be broken. I think this would be very rare but something worth discussion; I think I'm supporting of throwing PNSE everywhere, but people might differ on going the aggressive path:  @danmoseley @ericstj @terrajobst thoughts?

Reply:
What is useful in S.D.C on Unix that wouldn't require libgdiplus?

Reply:
I'd have to look at that. I can't think from top of my mind, but there might be some to just carry data (as data holders) without any data manipulation.

Reply:
On non-Windows platforms I set a dummy `DllImportResolver` that fails with that PNSE when a library is trying to be loaded from SDC.